package.json

```diff
@@ -1,93 +1,54 @@
 {
   "name": "module-skeleton",
-  "private": true,
```

Comment:
private true is needed
for non publishing root package

Reply:
sorry, it's unnecessary change